src/util.c

```diff
@@ -890,6 +890,23 @@ sds makePath(char *path, char *filename) {
     return sdscatfmt(sdsempty(), "%s/%s", path, filename);
 }
 
+int includeSpace(char *s) {
+    if (s == NULL) return 0;
+    for (size_t i = 0; i < strlen(s); i++) {
+        if (isspace(s[i])) {
+            return 1;
+        }
+    }
+    return 0;
+}
+
+char *rstrstr(const char *haystack, const char *needle) {
```

Comment:
a quick comment can help here
```suggestion
/* Reverse strstr (returning pointer to the last matching string or NULL). */
char *rstrstr(const char *haystack, const char *needle) {
```